src/Modal.svelte

```diff
@@ -106,6 +106,12 @@
    */
   export let closeOnEsc = true;
 
+  /**
+   * Scrolling behaviour after modal closes
+   * @type {"instant" | "smooth"}
```

Comment:
What about 
```suggestion
   * @type {'instant' | 'smooth' | 'auto'}
```

According to https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo, `auto` is the default value.